### PR TITLE
ENH: Make memory allocation more aggressive on local

### LINKED ISSUE #503: Insufficient memory is reported when 300MB(~30 mrow) of data is loaded into memory
The code is copied from this [blog post](https://xorbits.io/blogs/nyc-taxi-analysis): 

```
import datetime
import xorbits.pandas as pd

trips = pd.read_parquet(
    'yellow_tripdata_2021',
    columns=[
        'tpep_pickup_datetime',
        'tpep_dropoff_datetime',
        'trip_distance',
        'PULocationID',
        'DOLocationID',
    ])
# Remove outliers.
trips = trips[(trips['tpep_pickup_datetime'] >= datetime.datetime(2021, 1, 1)) & (trips['tpep_pickup_datetime'] <= datetime.datetime(2021, 12, 31))]
trips

```

The output of the program:

aad09505-a779-4732-b809-0db5df159d01

Environment: MacBook Air (16GB Memory)

The `htop` command on my mac shows:

image

### PR DESCRIPTION
**Pull request by aresnow1**
## What do these changes do?

Set hard limit to None for local deployment so that `QuotaActor` won't check available memory when allocate memory, it will reduce false alarms about insufficient memory.

## Related issue number

Fixes #503 

## Check code requirements

- [ ] tests added / passed (if needed)
- [ ] Ensure all linting tests pass

### CODE REVIEW

**Review by qianduoduo0904 — APPROVED** [review]
LGTM

File: python/xorbits/_mars/services/scheduling/worker/quota.py [hunk 1]
```diff
@@ -415,14 +415,22 @@ async def __post_create__(self):
...
+                    **band_config,
+                    uid=MemQuotaActor.gen_uid(band[1]),
+                    address=self.address,
+                )
+            else:
+                self._band_quota_refs[band] = await mo.create_actor(
+                    QuotaActor,
+                    band,
+                    quota_size=band_config["quota_size"],
+                    uid=MemQuotaActor.gen_uid(band[1]),
```
Comment (UranusSeven) [inline python/xorbits/_mars/services/scheduling/worker/quota.py:431]: Should this be `QuotaActor.gen_uid(band[1])`?

File: python/xorbits/_mars/deploy/oscar/local.py [hunk 2]
```diff
@@ -217,6 +217,11 @@ def __init__(
             oscar_extra_conf=oscar_extra_conf,
         )
 
+        # modify scheduling config to make memory allocation policy
+        # more aggressive on local
```
Comment (UranusSeven) [inline python/xorbits/_mars/deploy/oscar/local.py:221]:
```suggestion
        # make memory allocation policy more aggressive on local by
        # assuming all the memory are available.
```

File: python/xorbits/_mars/deploy/oscar/local.py [hunk 3]
```diff
@@ -217,6 +217,11 @@ def __init__(
             oscar_extra_conf=oscar_extra_conf,
         )
 
+        # make memory allocation policy more aggressive on local by
+        # assuming all the memory are available.
```
Comment (UranusSeven) [inline python/xorbits/_mars/deploy/oscar/local.py:221]:
```suggestion
        # assuming all the memory is available.
```

**Review by UranusSeven — APPROVED** [review]
LGTM.

**Pull request merged**